papaparse.js

```diff
@@ -1385,6 +1389,21 @@
 				lastCursor = cursor;
 			}
 
+			/**
+             * checks if there are extra spaces after closing quote and newLine without any text
+             * if Yes, returns the number of spaces
+            */
+            function extraSpacesLengthAfterQuote(index) {
+                var spaceLength = 0;
+				if (index !== -1) {
```

Comment:
Which caracter are you using for ident? It not rendered correctly on github, but you should use Tabs to ident code. I will expect this line to be aligned with the one upper.

Reply:
Using tabs but IDE might have messed it up. will fix it.